onnx/optimizer/passes/extract_constant_to_initializer.h

```diff
@@ -0,0 +1,46 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
```

Comment:
Nit: add pseudo code to explain the effect of the optimization pass.

Something like the following code:
https://github.com/onnx/onnx/blob/master/onnx/optimizer/passes/eliminate_unused_initializer.h#L7